syntax = "proto3";


enum ClassTypeEnum {
    UNKNOWN_CLASS = 0;
    CLASS_MONTESSORI = 1;
    CLASS_NURSERY = 2;
    CLASS_LOWER_KG = 3;
    CLASS_UPPER_KG = 4;
    CLASS_PREP = 5;
    CLASS_FIRST = 6;
    CLASS_SECOND = 7;
    CLASS_THIRD = 8;
    CLASS_FORTH = 9;
    CLASS_FIFTH = 10;
    CLASS_SIXTH = 11;
    CLASS_SEVENTH = 12;
    CLASS_EIGHTH = 14;
    CLASS_NINETH = 15;
    CLASS_TENTH = 16;
    CLASS_ELEVENTH = 17;
    CLASS_TWELVETH = 18;
}
